flight/PiOS/Common/pios_port.c

```diff
@@ -0,0 +1,623 @@
+#include <pios.h>
```

Comment:
never mind - looks like namespace in file is PIOS_HAL which is reasonable. can the file name match that?